packages/bootstrap/scss/core/color-system/_swatch-legacy.scss

```diff
@@ -116,7 +116,7 @@ $kendo-component-text: $kendo-body-text !default;
 /// @group component
 $kendo-component-border: $kendo-app-border !default;
 /// The background of the components' chrome area.
-$kendo-base-bg: if($kendo-enable-color-system, k-color(light), $kendo-color-light) !default;
+$kendo-base-bg: if($kendo-enable-color-system, k-color(surface), $kendo-color-light) !default;
```

Comment:
Maybe I am missing something, however In the _swatch.scss file _surface_ and _light_ colors have the same values 

https://github.com/telerik/kendo-themes/blob/develop/packages/bootstrap/scss/core/color-system/_swatch.scss#L108